app/pages/project/components/ProjectNavbar/components/ProjectNavbarWide/ProjectNavbarWide.jsx

```diff
@@ -1,35 +1,36 @@
-import React, { PropTypes } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import Avatar from '../Avatar';
 import Background from '../Background';
 import NavLink from '../NavLink';
 import ProjectTitle from '../ProjectTitle';
 import StyledHeader from '../StyledHeader';
 import Wrapper from '../Wrapper';
-import { colors, pxToRem } from '../../styledHelpers';
+import { pxToRem } from '../../../../../../theme';
 
-const StyledHeaderWide = StyledHeader.extend`
+export const StyledHeaderWide = StyledHeader.extend`
   box-shadow: 0 ${pxToRem(2)} ${pxToRem(4)} 0 rgba(0,0,0,0.5);
 `;
 
-const StyledAvatar = styled(Avatar)`
+export const StyledAvatar = styled(Avatar)`
   margin-right: ${pxToRem(20)};
 `;
 
-const StyledWrapper = styled(Wrapper)`
+export const StyledWrapper = styled(Wrapper)`
   box-sizing: border-box;
   min-height: ${pxToRem(150)};
   padding: 0 ${pxToRem(10)};
 `;
 
-const Nav = styled.nav`
+export const Nav = styled.nav`
   display: flex;
   flex: 1;
   justify-content: flex-end;
   margin-left: ${pxToRem(20)}
 `;
```

Comment:
not sure if want to add `flex-wrap: wrap` for unlikely edge cases with a ton of links like https://styled-components-theme.pfe-preview.zooniverse.org/projects/markb-panoptes/nfnorgtest-moths?facelift=true

Reply:
@beckyrother this is an interesting edge case. We do not limit the number of external links a project can add, so this could come up. One option is to wrap the links like Mark suggests:
<img width="1257" alt="screen shot 2018-01-31 at 2 02 38 pm" src="https://user-images.githubusercontent.com/5016731/35644773-8b5790d0-068f-11e8-90f9-f9b4a8b07677.png">

Or collapse the menu into the mobile style like we do with small view ports:
<img width="1418" alt="screen shot 2018-01-31 at 2 04 08 pm" src="https://user-images.githubusercontent.com/5016731/35644858-c3030190-068f-11e8-8547-4a54abd8f8b0.png">

I think collapsing it if it's larger than the max-width of 1200px is the preferable options since we don't know much larger it is.

Reply:
An interesting edge case indeed! I'm hesitant to say wrap everything together because the majority of our users still come to the site from desktop so they may not be familiar with the dropdown menu to navigate between Talk and Classify, for example. Is it possible to wrap just the additional external links?

Reply:
Good point. We do keep track of which links are external with a boolean in the code, so that might be doable.

Reply:
Nice! In that case maybe the dropdown still says "Explore Project" ?

Reply:
@beckyrother I think we'll have to go with a simple wrap like in the screenshot I took already. I tried separating out the internal and external links into their own navs and wrapped them into their own rows, which looked good on the desktop view, but it broke the dynamic math for shifting into the mobile style menu and caused overflow issues. This is probably not unsolvable, but I'm making a call that it's not worth the time investment since I think this will be a rare edge case.

Reply:
Sounds good. Thanks for looking into that. Is there any way to compromise and keep the title of the project over to the left as it is in the top screenshot, and move the drop-down menu to align right?

Reply:
I will look into it. It basically another layout variant, so shouldn't be too hard to put together.

Reply:
I got the title on the left and drop down menu on the right for this edge case: https://styled-components-theme.pfe-preview.zooniverse.org/projects/markb-panoptes/nfnorgtest-moths?facelift=true

Reply:
Perfect. Thank you!